### DISCUSSION TITLE
求个激活码

### DISCUSSION THREAD

**Discussion by TaixuJiang** [LFTEC/LearningFucker#45] [General]
YaezUZttmVkTkWRYJEe3FBYlo7UYh5/27MbmuNdU+oGzK3vRkb2Q9IuX3aXU2u1JbDA iJdUdnve+sb1g7kkmdmw+8Q0nAboJRWSrZh/NdhffUfo+6mO1b0t9E0wmLTawXfMDed 78GYG7M1TMGyk3otoe3q6Cj9sayMxnGZD1smvdmtbVRbwRb/xT+ekwBWUDZ1HD5l7fx ZT4d5cuCRCxrqFuyWDn4tXU8+hppFcG9O1UnBushLbr2MZVhZF50FDUaTIB8MKff+PN b1W5Dum/pHswbUoKeTyUfkQm8ZV433HDFBddrbJJ+F4RAJaBMwRz4//AApwAhACEeny FSxH4ecnjGvnsAR+lNcqU5wbkFbKudmAZy7M+jHiyYizKQ1nYsk7ffE1lyFNQzRU0Rb b0e2hNIHDsLHk7HKuUUDufjPS10BTjiaOuBSayCUOZ0EAPDpmJf1FGAUe9WoyYa4vsu Ma+WZM/tinUetWnU5+7StJDJQJUIbWwRiUk/gSzEC6Mua02NF+sL0K6aFpOOyeAIqjB jFsGJjwDF3fWMstD6rrnFXaMLMB2YsgAIp/IIK1U+LdRMlNC7zMFJDClD53wAhg59oC LwN8/iccdxxhqw+P6cnmmGYtDci4Mqz0dL5suR6kg4NgG97xHqUOsBN637zQ8g0p3jt xxyFPALC3+SmmfKgPI2LrBmLi7uGtS4C2uakB8FA2zPyGC1qCD0EZ62G2wuUurBqpFP f8F476K2oh3aKdLS/jknb9JHT9VnRw3jLT/G3wbWIZgGQg7/h0VJwNgvwfT6Fpam0ih 9J76Z2TqxtZ0/zRqV2TKHgrRtBfw4lLbgIkl+hYLG78IilohdciMx3pOxU/5eYRvdhW XSO3D/nV6IdJ255fcXkvqh/P+IuiXynDBibktD0MIFzetNk2ui5vCwzuTSQTr9aVpq9 A60bDvBsgd/K/8St25HVzsw54ANwCYEQMBNaGT8Anlb7mTQ4FkjU9aZ2dDsKVP5DTp5 HobmwwulLcEmYLnikzXb1+LKu5sF2NS++oVjeGKS4CH1/sJMvWiPlx5sBUIBZkOZ43/ OJipqU2PW0TF3YKP92XhWX89MyHrbONZehXZfiSJ8y7jYXEBU78aJHogxFXFmV8t8/z 2wZr/wG4+fDre9RfXII7rqgfIbSCaxag3BvT0wynKWD45JKJwWapUmW6XXPm/7jZ+Mg c6NZWjmXIyfJrq3qfgZPjJFZyzSJ17lQYKeAQ8jFnonfyhNFcX9g8s2NAG8nat9kL+R PuuuPcmy//9C6On4HC58bemdjVeAaS6rFY/NNaW49ESWLzqh2sFVeApoEYU6zJhNrcq TmQicC3V1HOyo1rjVy4g+QHD1Q==

**Comment by LFTEC** [comment]
Hh7lV1alAf9L8xBwO8v0jhxzzrPpMbGw5Cimy7JQh9Z+XSGPClLkJI6RCP+JbEoKv1VAcEfsRCRAXFQrDLSaKqPw9KEAOJ8A0iZHqCU7+NC7OpgPxLnnGkDDWml1a7AOfuQFS/LC1Rs116878axvKP40MtOIPumC7AscEnA5D97UJYH02wW/1fOKgJmnEEgqwsalffiLcM6kaSgoFHhGTw0U241/sPpnulHW1WjsvwW0tRMjl8nFgginc443v1R6aukpsMfuEJdt7L+SgEmjVsqWFj+PbxC6O3DPmsqQkypobh2S1X0uY4w4Y4OjkVVR35/DirBFwnBIYHZi4d0UiEXQ2EuXq/fTKmPSlFYBXUGhiPgTUd2bJFJkAZbcSpNoGuyGvsOKTlQUjIri80i3ofKTB1f7BBDSpPRpy6/BUqnf/m+FTpBUDnwWMi13ye1hFLjJFj+qqGTei/OYmc8j7lG1Ytaq8IIaqtnGKeyePQhKw4cTV8qcKdSMD+22B+N/FL5E6cS4+QdE6vYNNB0JY1WeraEYsoJEWFt9SwMqHoCBu9qHyC6CGUpE9e/5EpNWslD02npLfI+N4Cjb8c0J9fEl57turG3RfUXa4Gz+aQfHkhBelhZDUVRKh9lZhq7HD1015/k1Otg/3862NcmcfXekfNDe818JpC4Gl5Qmw4ZkQuAFrZEZSpKXEzSFbLWv4lmnJaYLVKkzVlbKHnPSNejR9nZBOPyNv+4hvvlfKuF5K7kDbM4VjmJW7kVoRc790BM3LT/yUm+g/QG4UBPsld1sIA+kqvEahIzPgTLSp44/U01zlMH3zR5j2GUt/QSyBPzy0sQ6E/gRKGMIvkffHT8iJzUysITOaMhW/e+SYSzSd28kZK0zMYTaKc5U5nRucpAFMJ3DnaqZSvUciujFo52eQT6B2uViT8IWC8xFQq8aS1OxFV1YutG+mfrUfXM8wFjvYtFZRlA9nxqyQfslXN33Y+QzTqbjVg1Nboq+hfK0RUAq+kl3XnaQ5XWC9+iRjNqCLfz52fK0tJPKiktvP2KSFEiG/NGk1Vp61MfmHxgAD1PadJcnBBmgkmyTOUyHc7HMCKxx4V5C3S8653mccQwW+fmmdEX4uKc1iciDAuqg0y6ryMWNYjlAZfs8uKngG9mWI5WXOosnx1oiVGFjLa1oVR+LR7grBF0C1Ewgmq+YL0HZTJ7ya3FscQ7LEuqE9fcH9gO3pzdMBpjR4n4cRjhwah7qdti0zOyZtkui3m21HoMyE6kGY+XMacBZXfra67nywYLvcccrPn7JEiWx5vBnnWxPKDlCF10T/aPmctHOlvIwSlz31kwamY055jFpPDltSgH0SvE5AOpnQgxPgQ==
